# feedback received
string feedback_received

# task being executed
string commanded_task
